### PR TITLE
Bash Script

### PR DESCRIPTION
**Pull request by jobysanthosh**

### CODE REVIEW

File: Borrower.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+sudo su
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+sudo yum update -y
+echo 'export SPRING_DATASOURCE_URL="jdbc:mysql://lms.cv2mc6y2qhct.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_USERNAME="admin"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_PASSWORD="*********"' >> /etc/environment
+
+export SPRING_DATASOURCE_URL="jdbc:mysql://lms.cv2mc6y2qhct.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"
```
Comment (janfernandosmoothstack) [inline Borrower.sh:11]: Do not post the link to the RDS.

File: Orchestrator.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+sudo yum update -y
+
+echo 'export SPRING_DATASOURCE_URL="jdbc:mysql://lms.cv2mc6y2qhct.us-east-2.rds.amazonaws.com:3306/users?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_USERNAME="admin"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_PASSWORD="**********S"' >> /etc/environment
+
+export SPRING_DATASOURCE_URL="jdbc:mysql://lms.cv2mc6y2qhct.us-east-2.rds.amazonaws.com:3306/users?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"
```
Comment (janfernandosmoothstack) [inline Orchestrator.sh:12]: Please remove link to RDS

File: Orchestrator.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+sudo su
```
Comment (janfernandosmoothstack) [inline Orchestrator.sh:2]: This line is not necessary.

File: Borrower.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+sudo su
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+sudo yum update -y
```
Comment (jobysanthosh) [inline Borrower.sh:6]: Work on having these statements inputted by another simple bash script for code re-usability and just being preloaded at bashrc or another bash profile

File: Bash.txt [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+
+echo 'export SPRING_DATASOURCE_URL="jdbc:mysql://lms.****.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_USERNAME="admin"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_PASSWORD="teamflash123"' >> /etc/environment
+export SPRING_DATASOURCE_URL="jdbc:mysql://lms.*****.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"
+export SPRING_DATASOURCE_USERNAME="***"
+export SPRING_DATASOURCE_PASSWORD="*****"
```
Comment (janfernandosmoothstack) [inline Bash.txt]: Space your script a little more so it is more readable.

File: lmsadmin.sh [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Janet Fernando
+# Admin User Data for AWS and LB 
+
+#!/bin/bash
+
+sudo yum install git -y
+sudo yum install java-1.8.0-openjdk -y
+sudo yum install maven -y
```
Comment (jobysanthosh) [inline lmsadmin.sh:8]: A simple script to store these first few commands would be nice in the future to be used for other EC2 instance User data

File: Borrower.sh [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+export SPRING_DATASOURCE_URL="jdbc:mysql://lms.cv2mc6y2qhct.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"
+export SPRING_DATASOURCE_USERNAME="admin"
+export SPRING_DATASOURCE_PASSWORD="*********"
+
+cd /home/ec2-user
+git clone https://github.com/sayanastha39/Borrower.git
+cd /home/ec2-user/Borrower/LMSBorrowerTry
+sudo mvn clean install
+nohup java -jar target/LMSBorrower-0.0.1-SNAPSHOT.jar &
```
Comment (jobysanthosh) [inline Borrower.sh:19]: Good job on using commands like nohup and '&' for the script

File: Bash.txt [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---------------------------MICRO SERVICES BASH SCRIPT--------------------------------
+#!/bin/bash
+sudo su
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+
+echo 'export SPRING_DATASOURCE_URL="jdbc:mysql://lms.****.us-east-2.rds.amazonaws.com:3306/library?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_USERNAME="admin"' >> /etc/environment
+echo 'export SPRING_DATASOURCE_PASSWORD="teamflash123"' >> /etc/environment
```
Comment (sayanastha39) [inline Bash.txt]: remove password

**Pull request closed without merging**